srcpkgs/gnome-core/template

```diff
@@ -10,27 +10,27 @@ homepage="https://www.gnome.org"
 
 depends="
  adwaita-icon-theme>=${version}
- at-spi2-core>=2.50.1
- at-spi2-atk>=2.50.1
+ at-spi2-core>=2.52.0
+ at-spi2-atk>=2.52.0
  cantarell-fonts>=0.301
  dconf>=0.40.0
- evolution-data-server>=3.50.4
- folks>=0.15.6
+ evolution-data-server>=3.52.0
+ folks>=0.15.8
  font-adobe-source-code-pro>=2.030R
  gcr>=3.41.1
```

Comment:
@oreo639 shouldn't we depend on `gcr4` instead?
all the other packages are switching to `gcr4-devel`

Reply:
Not entirely sure why gcr is depended on by gnome-core tbh. Currently we only provide the gcr-ssh-agent executable with gcr3, although that can probably be changed.

imo, leave it for now, although it will probably be removed later.